docs/playbook.md

```diff
@@ -18,6 +18,26 @@ Please consult the [flags.go](https://github.com/GoogleCloudPlatform/gcsfuse/blo
 ### Mount successful but files not visible
 This is explained in the [semantics](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#implicit-directories) page and can possibly be fixed by passing the --implicit-dirs=true flag to the gcsfuse command.
 
+### Mounting Bucket doesn't get Succeeded
+
+While running
+
+* gcsfuse --implicit-dirs <bucket> ~/gcsfuse
```

Comment:
also this is not required. mounting heading itself means you are invoking gcsfuse mount command. Just mention as follows

The error comes when bucket is already mounted in a folder and we try to mount it again in the same folder without un-mounting it. Fix it to first umount and then remount. Command for un mounting. 
sudo umount ~/gcsfuse